sources/Debate/src/main/java/com/example/Debate/service/ArgumentServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.example.Debate.service;
+
+import com.example.Debate.dto.ArgumentDto;
+import com.example.Debate.model.Argument;
+import com.example.Debate.repository.ArgumentRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class ArgumentServiceImpl implements ArgumentService{
+    private ArgumentRepository argumentRepository;
+    private ModelMapper modelMapper;
+
+    public ArgumentServiceImpl(ArgumentRepository argumentRepository, ModelMapper modelMapper) {
+        this.argumentRepository = argumentRepository;
+        this.modelMapper = modelMapper;
+    }
+
+    @Override
+    public ArgumentDto getArgumentById(String id) {
+        Argument argument = argumentRepository.findById(id).get();
```

Comment:
Usage of optional classes could be better - e. g. argumentRepository.findById(id).orElseThrow(...). We will work on that in the next sprints.